vehicle-model-lifecycle/src/download_vspec.py

```diff
@@ -128,7 +137,14 @@ def get_vehicle_signal_interface_src(interface: Dict[str, Any]) -> str:
     Returns:
         str: The URI of the source for the Vehicle Signal Interface.
     """
-    return str(interface["config"]["src"])
+    src = ""
+    unit_src_list = []
+    if "config" in interface:
+        if "src" in interface["config"]:
+            src = str(interface["config"]["src"])
```

Comment:
please add 2 variables into: https://github.com/eclipse-velocitas/devenv-devcontainer-setup/blob/main/manifest.json#L78

1. vssSrc (default "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json")
2. vssUnitSrc (default [                    "https://github.com/COVESA/vehicle_signal_specification/blob/v4.0/spec/units.yaml"])

after that start assign "src" and "unit_src_list" those variables as a default -> require_env("vssSrc) & require_env("vssUnitSrc)